documentation/book/ref-topic-operator-usage-recommendation.adoc

```diff
@@ -5,11 +5,28 @@
 [id='topic-operator-usage-recommendations-{context}']
 = Topic Operator usage recommendations
 
-* Be consistent and always operate on `KafkaTopic` resources or always operate on topics directly. Avoid routinely using both methods for a given topic.
-* When creating a `KafkaTopic` resource:
-** Remember that the name cannot be changed later.
-** Choose a name for the `KafkaTopic` resource that reflects the name of the topic it describes.
-** Ideally the `KafkaTopic.metadata.name` should be the same as its `spec.topicName`. To do this, the topic name will have to be a link:https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture/identifiers.md[valid {KubernetesName} resource name^].
-* When creating a topic:
-** Remember that the name cannot be changed later.
-** It is best to use a name that is a link:https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture/identifiers.md[valid {KubernetesName} resource name^], otherwise the operator will have to modify the name when creating the corresponding `KafkaTopic`.
+When working with topics, be consistent and always operate on either `KafkaTopic` resources or topics directly. Avoid routinely switching between both methods for a given topic.
+
+Use topic names that reflect the nature of the topic, and remember that names cannot be changed later.
+
+If creating a name directly for a topic, use a name that is a valid {KubernetesName} resource name, otherwise the Topic Operator will need to modify the name when creating the corresponding `KafkaTopic`.
+
+.Kafka topic naming conventions
+
+Kafka and {KubernetesName} impose their own validation rules for the naming of `KafkaTopic` and `KafkaTopic.metadata.name` respectively.
+So there are valid naming conventions for each which are invalid in the other.
```

Comment:
Can we link to the rules directly here and not only at the end in additional resources section?

Reply:
I hate `Additional Resources`. Who thinks to look there when reading the documentation as a web page? We expect such references to be hyperlinks inline with the text.

Reply:
Right. I personally don't mind having them there. But the links should be always also in the text.

Reply:
I guess the idea with Additional resources is to link to content outside the current assembly. And to encourage the reader to concentrate on the content in front of them without navigating away. I agree that it can seem a bit cluttered in the web format. I don't think we need links in the content and an Additional resources in this context, so removed Additional resources.

Reply:
Well, I think that if you don't want the link there then you need to briefly explain the rules in the text. Because it is not useful to treat the reader as _there are some restrictions but we will not tell you which_. The additional resources are not useful here because the user will not have them when he reads this sentence.